cdf_fabric_replicator/time_series.py

```diff
@@ -136,7 +136,7 @@ def send_time_series_to_lakehouse_table(self, subscription: SubscriptionsConfig,
         metadata = ts.metadata if len(ts.metadata) > 0 else {"source": "cdf_fabric_replicator"}
         df = pd.DataFrame(np.array([[ts.external_id, ts.name, ts.description if ts.description else "" , ts.is_string, ts.is_step, ts.unit, metadata, asset_xid]]), columns=["externalId", "name", "description", "isString", "isStep", "unit", "metadata", "assetExternalId"])
         df = df.dropna()
-        self.logger.info (f"Writing {ts.external_id} to '{subscription.lakehouse_abfss_path_ts}' table...")
```

Comment:
I initially thought this was a bug but it looks like I just missed setting up the logger in the replicator.  However, the logger from self isn't used anywhere else in the code, all the other logging statements are through the `logging` library.  That's why I thought this was a bug, since it didn't match how all the other logging statements were being made

Reply:
I changed this back and am using a mock for the logger, but we should still standardize how we log across the replicator